starknet-foundry/crates/forge/src/scarb.rs

```diff
@@ -338,7 +346,7 @@ mod tests {
             .exec()
             .unwrap();
 
-        let (_, _, dependencies, _) =
+        let (_, _, _, dependencies, _) =
```

Comment:
I think we should create an issue to verify if other values than just `dependencies` are set correctly by these methods.

Reply:
Done